assets/js/components/contacts.js

```diff
@@ -109,6 +118,21 @@ export class ContactSection {
         .css("display", "inline");
     }
 
-
+    this.$staffLinks = this.$contactContainer.find(".expand-block__trigger a");
+    this.$muniLinks = this.$element.find("a");
+    this.$staffLinks.on('click', ((e) => {
+      gtag('event', 'contact_info', {
+        category: "Contacts",
+        action: "Click",
+        label: `${$(e.target).text()}`,
+      });
+    }));
+    this.$muniLinks.on('click', ((e) => {
+      gtag('event', 'contact_info', {
+        category: "Contacts",
+        action: "Click",
+        label: `Municipal wide contacts`,
```

Comment:
Don't need `` if no variable interpolation.

Reply:
Changed literal to regular string